whelk-core/src/main/resources/ext/marcframe.json

```diff
@@ -17508,12 +17548,13 @@
       ]
     },
     "773": {
-      "include": ["referenceToWork"],
-      "aboutEntity": "?work",
-      "aboutAlias": "_:work",
+      "include": ["referenceToInstanceSansDisplayText"],
+      "aboutEntity": "?thing",
+      "aboutAlias": "_:instance",
       "addLink": "isPartOf",
-      "resourceType": "Aggregate",
+      "resourceType": "Instance",
```

Comment:
The global change needs to take this into account as well. I.e. change all uses of `"@type": "Aggregate"` to `"Instance"`.

Reply:
Hm, maybe it (implicitly) already do so with the newInstanceObject?

Reply:
Yes it does. Before adding to newInstanceObject, the script sets type to 'Instance' (line 125). The same applies to dataSources that used to have type 'Dataset '